modules/ztp-du-cluster-config-reference.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-vdu-configuration-reference.adoc
+
+:_module-type: REFERENCE
+[id="ztp-du-cluster-config-reference_{context}"]
+= Recommended cluster configurations to run vDU applications
+
+Clusters running virtualized distributed unit (vDU) applications require a highly tuned and optimized configuration. The following information describes the various elements that you require to support vDU workloads in {product-title} {product-version} clusters.
+
+[NOTE]
+====
+You manage your custom site configuration data in a Git repository accessible from the hub cluster that is defined as a source repository for Argo CD. You extract the baseline source custom resources (CRs) from the zero touch provisioning (ZTP) container.
+====
+
+== Recommended cluster MachineConfig installation CRs
```

Comment:
Minor note... I would leave out the "installation" qualifier. It implies these are somehow different than typical MachineConfig CRs. (3 total places in this section)